data_host/chrome_extension/src/pages/Popup/components/Form/index.tsx

```diff
@@ -3,6 +3,10 @@ import { useDispatch } from 'react-redux';
 import { Form as FinalForm, Field, FormRenderProps } from 'react-final-form';
 import { FormApi } from 'final-form';
 
+import Error from '../../../../components/Error';
```

Comment:
I'm generally not a fan of importing from relative paths for parent directories. If we move this file and not the other one, this import needs to change. it's also really unclear where this is coming from. 

Are we able to import from src/components/Spacing instead? If not, can we change some config so we can?

Reply:
Just pushed configuration including path aliases 👍